src/test/scenarios/managed-network/output/src/managednetwork/azext_managednetwork/generated/_help.py

```diff
@@ -264,10 +264,10 @@
       - name: Create/Update Managed Network Peering Policy
         text: |-
                az managednetwork managed-network-peering-policy mesh-topology create --managed-network-name "myManagedN\
-etwork" --policy-name "myHubAndSpoke" --type "HubAndSpokeTopology" --hub id="/subscriptions/subscriptionB/resourceGroup\
-s/myResourceGroup/providers/Microsoft.Network/virtualNetworks/myHubVnet" --spokes id="/subscriptions/subscriptionB/reso\
-urceGroups/myResourceGroup/providers/Microsoft.ManagedNetwork/managedNetworks/myManagedNetwork/managedNetworkGroups/myM\
-anagedNetworkGroup1" --resource-group "myResourceGroup"
+etwork" --policy-name "myHubAndSpoke" --hub id="/subscriptions/subscriptionB/resourceGroups/myResourceGroup/providers/M\
+icrosoft.Network/virtualNetworks/myHubVnet" --spokes id="/subscriptions/subscriptionB/resourceGroups/myResourceGroup/pr\
+oviders/Microsoft.ManagedNetwork/managedNetworks/myManagedNetwork/managedNetworkGroups/myManagedNetworkGroup1" --resour\
+ce-group "myResourceGroup"

```

Comment:
the example should still be only applicable to the hub-and-spoke-topology not the mesh-topology